### ISSUE TITLE
Enhancement: Option to choose to open tab in background as default

### ISSUE THREAD

**Issue by AtTheGates** [tiansh/simple-context-search#4]
Hello,

first thank you very much for your great and lightweight extension.

I know, opening tabs in background is possible with the Ctrl key. But I would love to see an option to open tabs in background as default, like the old "Context Search" extension by Ben Basson did. Would that be possible?

Thank you very much in advance. :-)

**Comment by tiansh** [comment]
I agree. It could be an option to switch de default action without <kbd>Ctrl</kbd> key. And maybe I will add it later. A PR is welcomed if someone want to work on this (just comment before you do so).

I _will_ work on this. But may be a few days / weeks later.
// Since i am being addicted to a new console game this week :)

**Comment by Haven667** [comment]
Hi,

I don't want to bother you, but are there any news on this? Thank you. :-)

**Comment by Haven667** [comment]
Thank you very much for the new version. :-) Would it be also possible to open tabs as last tab (right) on the tab bar? That would make it perfect for me, like the old "Context Search" extension by Ben Basson.
